centreon/src/Security/Domain/Authentication/AuthenticationService.php

```diff
@@ -143,7 +144,7 @@ public function findAuthenticationTokensByToken(string $token): ?AuthenticationT
     /**
      * @inheritDoc
      */
-    public function updateAuthenticationTokens(AuthenticationTokens $authenticationTokens): void
+    public function updateAuthenticationTokens(SecurityAuthenticationTokens $authenticationTokens): void
```

Comment:
The **AuthenticationService** class implements the **AuthenticationServiceInterface** interface, and the signature of the **updateAuthenticationTokens** method is as follows:

public function updateAuthenticationTokens(Core\Security\Authentication\Domain\Model\AuthenticationTokens $authenticationToken) : void ;

Your implementation is different

Reply:
yeap i m using SecurityAuthenticationTokens (Security\Domain\Authentication\Model\AuthenticationTokens) because that fixing the problem in AuthentificationRepo that why i change the signature